static/src/js/components/SecondaryToolbar/SecondaryToolbar.jsx

```diff
@@ -87,6 +89,7 @@ class SecondaryToolbar extends Component {
     onUpdateProjectName(newProjectName)
   }
 
+  // Needed for Save Project so page does not change
```

Comment:
Updated. we discussed this at one point as a team if we needed some of the `event.stopPropagation()` calls but, we do because if we don't have that when the project-id updates the url path it tries to refresh the page. (If you remove them I mean)